Tests/tests/test_FrontPanel.cpp

```diff
@@ -0,0 +1,820 @@
+#include <gtest/gtest.h>
+
+#include "FrontPanel.h"
+#include "frontpanel.h"
+
+//required to set initDone to 0, so we reset values in CFrontPanel instance.
+#include "frontpanel.cpp"
```

Comment:
there are better ways e.g. make initDone a public member or make it externally linkable

Reply:
There is no need for frontpanel.cpp, was left over from what I was trying to figure out the frontpanel memory leak issue that we've decided to not worry about. HAs been removed.